content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/clientlibs/site/js/image.js

```diff
@@ -197,6 +197,16 @@
             });
 
             that._elements.image.addEventListener("cmp-image-redraw", that.update);
+
+            var interSectionObserver = new IntersectionObserver(function(entries, interSectionObserver) {
+                entries.forEach(function(entry) {
+                    if (entry.intersectionRatio > 0) {
+                        that.update();
+                    }
+                });
+            });
+            interSectionObserver.observe(that._elements.self);
```

Comment:
Should we disconnect the observer once the image got initialised? 

Otherwise the observer would still be called even if there is nothing to do in `this.update()`

Reply:
fixed in 47f7cbaa0d896ca1efe7579fc9e95e2ccb0f75dd